admin_apps/README.md

```diff
@@ -1,35 +1,37 @@
-# Admin apps
+# Admin App
 
-This folder contains a Streamlit app designed that is designed to help you iterate your semantic models for the Cortex Analyst feature.
-
-The iteration app currently points to an existing semantic model on stage, allow you to chat against it, test the result and modify for verified queries.
+This folder contains a Streamlit app designed that is designed to help you create and iterate on your semantic models
+for the Cortex Analyst feature.
 
+The app currently supports two types of flows:
 
 <table border="0">
    <tr>
      <td><img src="https://github.com/Snowflake-Labs/semantic-model-generator/assets/103658138/2b5fd664-9a9a-46ed-afbf-cb7638f0ad19" width="700"></td>
```

Comment:
Will update screenshots in a separate PR so that I can properly upload them to the correct remote Github source.